### ISSUE TITLE
AsyncChatEvent 

### ISSUE THREAD

**Issue by contti228** [Aust1n46/VentureChat#122]
The following plugins have listener in legacy org.bukkit.event.player.AsyncPlayerChatEvent event: VentureChat.
[16:43:34 WARN]: [SignedVelocity] This may negatively affect the functionality of SignedVelocity,
[16:43:34 WARN]: [SignedVelocity] please report to the author to use Paper's AsyncChatEvent and/or AsyncChatDecorateEvent.

**Comment by Lia-LV** [comment]
True. Currently, the AsyncChatEvent is not detected properly for chats using VentureChat. please update ChatEvent to Paper's AsyncChatEvent (when server using Paper or it's fork)

**Comment by contti228** [comment]
I made a few changes to the plugin and now it uses AsyncChatEvent. do you know when the ability to delete messages will be added? I tried to do something myself, but it didn't work =(
